lighthouse-core/report/v2/renderer/category-renderer.js

```diff
@@ -122,6 +122,43 @@ class CategoryRenderer {
    * @param {number} scale
    * @return {!Element}
    */
+  _renderTimelineMetricAudit(audit, scale) {
+    const element = this._dom.createElement('div', [
+      'lh-timeline-metric',
+      `lh-timeline-metric--${Util.calculateRating(audit.score)}`,
+    ].join(' '));
+
+    const sparklineContainerEl = this._dom.createChildOf(element, 'div',
+        'lh-timeline-metric__sparkline');
+    const titleEl = this._dom.createChildOf(element, 'div', 'lh-timeline-metric__title');
+    const titleNameEl = this._dom.createChildOf(titleEl, 'span', 'lh-timeline-metric__name');
+    const titleValueEl = this._dom.createChildOf(titleEl, 'span', 'lh-timeline-metric__value');
+    titleNameEl.textContent = audit.result.description;
+    titleValueEl.textContent = audit.result.displayValue;
```

Comment:
this is controlled by the audits right now, should we ignore displayValue and just handle here or fix the display values?